netbsd/Platform.c

```diff
@@ -512,8 +512,9 @@ void Platform_getBattery(double* percent, ACPresence* isOnAC) {
          *isOnAC = isConnected ? AC_PRESENT : AC_ABSENT;
       }
    }
-
-   *percent = ((double)totalCharge / (double)totalCapacity) * 100.0;
+   *percent = (totalCapacity > 0)
+    ? ((double)totalCharge / (double)totalCapacity) * 100.0
+    : NAN;
```

Comment:
Reasonably long lines are preferred.
```suggestion
   *percent = totalCapacity > 0 ? ((double)totalCharge / (double)totalCapacity) * 100.0 : NAN;
```

Technically you could even do:
```suggestion
   *percent = totalCapacity > 0 ? (100.0 * totalCharge) / totalCapacity : NAN;
```

Please squash commits. The commit history should tell what needed to change, not how you got there.

Reply:
@BenBE Your latter suggestion changes semantics. I would rather not write that way. For floating points, division before multiplication is safer against overflow, and the rounding differences in the fractions part is negligible.

Reply:
The latter suggestion is the order used on Darwin and Linux. For OpenBSD the division is performed first.